reddit/reddit.go

```diff
@@ -109,3 +112,43 @@ func MaxFeedForCtx(ctx context.Context) int {
 
 	return GuildMaxFeedsNormal
 }
+
+func CheckSubreddit(data string) (bool, error) {
+	var redditStruct struct {
+		Data struct {
+			Dist     int `json:"dist"`
+			Children []struct {
+			}
+		} `json:"data"`
+	}
```

Comment:
Perhaps name this `redditData`? Generally naming variables with their types (Hungarian notation) is discouraged (e.g., you didn't use `dataString` as the name of the first parameter.)